openrtx/include/ui/ui_default.h

```diff
@@ -224,6 +230,7 @@ typedef struct ui_state_t
     char new_time_buf[9];
 #endif
     char new_callsign[10];
+    char new_message[822];
```

Comment:
This state is specific to new m17 meta text data, so how about we rename it to `m17_new_meta_text_data`?